docs/getting-started/advanced/backups.md

```diff
@@ -8,10 +8,21 @@ A full backup of your PhotoPrism instance includes:
 The best way to create an index backup is to run this command in a terminal:
 
 ```
-photoprism backup -a -i [filename]
- ```
+photoprism backup -i [filename]
+```
+
+Or the following for [docker-compose](../docker-compose.md):
+
+```
+docker-compose exec photoprism photoprism backup -i - > photoprism-db.sql
```

Comment:
It's important to use the -T parameter when running the command via docker-compose as otherwise debug, info, and warning logs will end up in the SQL dump too:
```
docker-compose exec -T photoprism photoprism backup - > photoprism-db.sql
```

Reply:
I'll add it later.